packages/feat-num/feat-num.20211224/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis:
+  "Facilities for enumerating and sampling algebraic data types, using Num for big numbers"
+maintainer: "francois.pottier@inria.fr"
+authors: "François Pottier <francois.pottier@inria.fr>"
+homepage: "https://gitlab.inria.fr/fpottier/feat"
+bug-reports: "francois.pottier@inria.fr"
+depends: [
+  "ocaml" {>= "4.03"}
+  "dune" {>= "1.3"}
+  "feat-core"
```

Comment:
```suggestion
  "feat-core" {= version}
```
it might be easier for users to keep track of subpackages if they’re synch’d up together